com.unity.netcode.gameobjects/Runtime/NetworkVariable/Collections/NetworkList.cs

```diff
@@ -425,7 +425,7 @@ public IEnumerator<T> GetEnumerator()
         public void Add(T item)
         {
             // check write permissions
-            if (!CanClientWrite(m_NetworkBehaviour.NetworkManager.LocalClientId))
+            if (m_NetworkBehaviour && !CanClientWrite(m_NetworkBehaviour.NetworkManager.LocalClientId))
```

Comment:
Seeing as `m_NetworkBehaviour` and  `CanClientWrite` are both defined on the `NetworkVariableBase`, our preferred solution in this area is to update the `CanClientWrite` to make the `clientId` argument optional. If no `clientId` is passed then the `LocalClientId` is used instead. That way we are not duplicating the `m_NetworkBehaviour` check.

Reply:
Updated to match what's done in NetworkVariable: https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/blob/c6f5c165dad9f065efc45298930c46b854668e9a/com.unity.netcode.gameobjects/Runtime/NetworkVariable/NetworkVariable.cs#L115